### PR TITLE
New netcat Cheat Sheet

### PR DESCRIPTION
**Pull request by biscuitNinja**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [x] Cheat Sheets: **`New netcat Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Addition of a cheat sheet for netcat

###### Which issues (if any) does this fix?

N/A

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/netcat_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @biscuitNinja

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by biscuitNinja** [comment]
Hmmm, of course the Travis build fails the first time around with:

"FAIL: No Instant Answer page found with ID 'netcat_cheat_sheet'"

Which comes first, the change or the pull request? ;)

**Comment by biscuitNinja** [comment]
I'm sorry I need to fix the tabbing... another commit to follow in a few moments.

File: share/goodie/cheat_sheets/json/netcat.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
+{
+    "id": "netcat_cheat_sheet",
+    "name": "Netcat",
+    "description": "Useful and commonly used netcat commands",
+    "metadata": {
+        "sourceName": "nc on OpenBSD.org",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/netcat.json]: " OpenBSD " would be enough here

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=netcat cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/netcat_cheat_sheet_mobile.png?xXzgNfis6J)

**Comment by daxtheduck** [comment]
@biscuitNinja this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by mintsoft** [comment]
Thanks @biscuitNinja

**Pull request merged**